### PR TITLE
Fix initiator and update package lock

### PR DESCRIPTION
**Pull request by dadon-david**
Hello,

I fixed the initiator, its now have different properties according to its type property:

**For all type:**
- detail (serialized version of the whole initiator object)
- type (script, parser or other)

**Parser type:** 
- url
- line number

**Script type:** 
- url _of the top element in the call frame_
- line number _of the top element in the call frame_
- column number _of the top element in the call frame_
- function's name _of the top element in the call frame_
- script's id _of the top element in the call frame_

And I updated the package.lock.json

Thanks 😄

### CODE REVIEW

**Comment by dadon-david** [comment]
Strange, `npm run lint` do not raise me any error on my machine, do I miss something ?

**Comment by dadon-david** [comment]
Also, I found that the CI has failed with the precedent commit https://travis-ci.org/sitespeedio/chrome-har/jobs/322701738 too.

Do I have to change something in the code or does the CI criteria (or the linter in the code) have to be change ?

Thanks 😄

File: index.js [hunk 1]
```diff
@@ -262,13 +262,39 @@ module.exports = {
...
+              case 'script':
+                {
+                  if (
+                    params.initiator.stack &&
+                    params.initiator.stack.callFrames.length > 0
+                  ) {
+                    const topCallFrame = params.initiator.stack.callFrames[0];
+                    entry._initiator = topCallFrame.url;
+                    entry._initiator_line = topCallFrame.lineNumber + 1; // Because lineNumber is 0 based
+                    entry._initiator_column = topCallFrame.columnNumber;
```
Comment (tobli) [inline index.js]: This is also 0-based, right?

**Comment by tobli** [comment]
Since we don’t lock down dev dependencies, the version of prettier that’s the current at the time will be run. There are sometimes small formatting changes to prettier, so a previously green build might shift to red, just by running npm install at a later time. Not ideal, we should probably lock it down, at least to a major version.

I can have a closer look, hopefully later tonight. Changes look awesome btw. =)

File: index.js (hunk 1 above)
Comment (dadon-david) [reply index.js]: Idk, will check 😄

Comment (dadon-david) [reply index.js]: Yes it's 0 based https://chromedevtools.github.io/devtools-protocol/1-2/Runtime#type-CallFrame

Will commit the fix

**Pull request merged**